src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs

```diff
@@ -462,6 +462,31 @@ private void SendViewAddedEventToWindow(IntPtr data)
             }
         }
 
+        private EventHandler _backKeyPressed;
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler BackKeyPressed
```

Comment:
I think it would be better to add some comments even though it is hidden API. ^^;

Reply:
DONE